src/utils/streams.js

```diff
@@ -0,0 +1,136 @@
+'use strict';
+
+const fs = require('fs');
+const fsPromises = require('fs').promises;
+const program = require('commander');
+const csvjson = require('csvjson');
+const axios = require('axios');
+
+// example url with css file for css bundler
+const url = 'https://gist.githubusercontent.com/eirikbakke/1059266/raw/d81dba46c76169c2b253de0baed790677883c221/gistfile1.css';
+
+// I/O functions
+const readStream = (filePath) => {
+  const read = fs.createReadStream(filePath, 'utf8');
+  read.on('error', err => console.log(`reading error: ${err}`));
+  return read;
+};
+
+const writeStream = (filePath) => {
+  const fileName = filePath.replace(/^.*[\\\/]/, '').slice(0, -4);
```

Comment:
Don't use regex here, use `Path` module and methods such as https://nodejs.org/api/path.html#path_path_dirname_path

Regex is unsafe and hard to maintain, especially for cross-platform software, where same code can be used under linux/mac/windows environment. `Path` module is cross-platform solution and much easier to use than to modify paths manually every time by regex or other methods.